index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/73a06aba35.js" crossorigin="anonymous"></script>
+    <title>Portfolio2</title>
+  </head>
+  <body>
+
+
+    <div class="container">
+      <header>
+        <a id="logo" href="index.html">Welcome</a>
+        <nav>
+          <button type="button" class="hamburger"><i class="fa-solid fa-bars hamburger"></i></button>
+        </nav>
+      </header>
+        
+
+      <section class="main-section">
+        <section class="primary-text">
+          <h1>Hey there.I'm<br>Hammed<br></h1> 
+          <h2>I'm a Software Developer</h2>
+        </section>
```

Comment:
- [ ] Please don't use `h2` tag here because `I'm a Software Developer` is also a part of your hero title :blush: Please use a single h1 tag for you hero title :+1:
**Fix**
```
<h1>
  Hey there, I'm<br>Hammed</br>
  <span>I'm a Software Developer</span>
</h1>
```